templates/activity.html

```diff
@@ -13,6 +13,7 @@
       <meta content="text/html">
   </head>
   <body>
+    {% include 'ribbon.html' %}
```

Comment:
I suspect `{{ link }}` in `ribbon.html` would not get translated, because you are not using `include ... with ...` grammar. Btw, your `ribbon/views.py` is not called at all. The naive solution is to copy that logic into every view, pass the link as context, and use `include ... with ...` grammar, but that brings a lot of redundancy.

https://stackoverflow.com/questions/21062560/django-variable-in-base-html looks like a solution.

Reply:
🤔

Reply:
Check now